docs/operator-manual/monitor-mode.md

```diff
@@ -1,44 +1,37 @@
 ---
-sidebar_label: "Monitor Mode"
-title: ""
+sidebar_label: "Monitor mode"
+title: "Monitor mode"
+description: The Kubewarden monitor mode and how to activate it.
+keywords: [kubernetes, kubewarden, monitor mode]
 ---
 
-# Monitor mode
+After you deploy a policy on the cluster it starts accepting, rejecting or mutating requests.
```

Comment:
I think we should  reword the explanation. I find it would be better to state something like:

When defining a policy, you can choose between two modes, specified in its spec.mode` (see https://docs.kubewarden.io/operator-manual/CRDs#admissionpolicyspec).
By default, the policy is deployed in mode `protect`, where it will accept, reject, or mutate request.

One can elect to deploy a policy in `monitor` mode. In `monitor` mode:

1. (cue to the numbered list below)

Reply:
Thanks.